components/admin_console/billing/plan_details.tsx

```diff
@@ -246,6 +246,12 @@ const PlanDetails: React.FC = () => {
             </div>
             {planPricing}
             {showSeatsAndSubscriptionDates && seatsAndSubscriptionDates(locale, userCount, subscription.seats, new Date(subscription.start_at), new Date(subscription.end_at))}
+            <div className='PlanDetails__description'>
+                <FormattedMarkdownMessage
```

Comment:
Doesn't need to be a markdown message, can just be a `FormattedMessage`

Reply:
Copy pasted. Fixed!